Configurations/ConfigFramework.xcconfig

```diff
@@ -1,8 +1,8 @@
 // Framework only
 
 DYLIB_INSTALL_NAME_BASE = @rpath
-DYLIB_COMPATIBILITY_VERSION = 1.6
-DYLIB_CURRENT_VERSION = $(SPARKLE_VERSION_MAJOR).$(SPARKLE_VERSION_MINOR).$(SPARKLE_VERSION_PATCH)
+DYLIB_COMPATIBILITY_VERSION = $(SPARKLE_VERSION_MAJOR).0.0
+DYLIB_CURRENT_VERSION = $(SPARKLE_VERSION)
```

Comment:
The dylib version is not supposed to contain $SPARKLE_VERSION_SUFFIX. `$(SPARKLE_VERSION_MAJOR).$(SPARKLE_VERSION_MINOR).$(SPARKLE_VERSION_PATCH)` was correct.

Reply:
Unless you meant for `SPARKLE_VERSION` to not include the suffix. I don't think this variable is used anywhere else.

Reply:
Nope, I got this wrong. I've reverted it in 85426bf03791c26c257c38e722df6b57125e7236.